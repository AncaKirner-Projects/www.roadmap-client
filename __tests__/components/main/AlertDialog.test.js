import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import AlertDialog from '../../../src/components/main/AlertDialog';

describe('<AlertDialog />', () => {
  const props = {
    open: true,
    title: 'Alert',
    text: 'Alert text',
    handleClose: jest.fn()
  };

  it('renders an element', () => {
    const wrapper = shallow(<AlertDialog {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
