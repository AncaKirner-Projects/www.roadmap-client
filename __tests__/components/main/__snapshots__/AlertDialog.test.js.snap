// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlertDialog /> renders an element 1`] = `
<WithStyles(Dialog)
  aria-describedby="alert-dialog-description"
  aria-labelledby="alert-dialog-title"
  open={true}
>
  <WithStyles(DialogTitle)
    id="alert-dialog-title"
  >
    Alert
  </WithStyles(DialogTitle)>
  <WithStyles(DialogContent)>
    <WithStyles(DialogContentText)
      id="alert-dialog-description"
    >
      Alert text
    </WithStyles(DialogContentText)>
  </WithStyles(DialogContent)>
  <WithStyles(DialogActions)>
    <WithStyles(Button)
      autoFocus={true}
      color="primary"
      onClick={[MockFunction]}
    >
      OK
    </WithStyles(Button)>
  </WithStyles(DialogActions)>
</WithStyles(Dialog)>
`;
